static/js/containers/PaymentPage.js

```diff
@@ -247,6 +247,7 @@ class PaymentPage extends React.Component {
             payment={payment}
             payableKlassesData={payableKlassesData}
             selectedKlass={selectedKlass}
+            now={moment()}
```

Comment:
Should this be generated in `componentWillMount` instead so we don't suddenly update the message if the day changes?